tests/onnxruntime/test_modeling_ort.py

```diff
@@ -357,15 +361,15 @@ def test_supported_transformers_architectures(self, *args, **kwargs):
 
     def test_load_vanilla_transformers_which_is_not_supported(self):
         with self.assertRaises(Exception) as context:
-            model = ORTModelForFeatureExtraction.from_pretrained("google/vit-base-patch16-224", from_transformers=Tru)
```

Comment:
the idea was here to use a "architecture" which we don't support with `from_transformers`

Reply:
Same here